Restructure documentation in test-infra (#110)
kyma-project/test-infra: prow/scripts/README.md

# Cluster

## Overview

The folder contains scripts involved in integration tests.

### Project structure

<!-- Update the folder structure each time you modify it. -->

The structure of the folder looks as follows:

```

>   ├── cluster-integration                # Scripts for the cleanup of Google Cloud Platform (GCP) objects after completion of integration tests.             

Reviewer: Also: dots

  ├── library.sh                         # The script to be used as an integral part of other scripts, for example by the "pipeline.sh" script. With proper parameters defined, it authenticates you to GCP and sets up the Docker-in-Docker environment.

>   ├── pipeline.sh                        # The script responsible for building and testing a given Kyma component by running the respective "Makefile" target.

Reviewer: ```suggestion
  ├── pipeline.sh                        # The script which builds and tests a given Kyma component by running the respective "Makefile" target.
```

>   └──  provision-vm-and-start-kyma.sh    # The script responsible for starting a virtual machine as part of the integration job and running integration tests for Kyma on Minikube.

Reviewer: ```suggestion
  └──  provision-vm-and-start-kyma.sh    # The script which starts a virtual machine as part of the integration job and runs integration tests for Kyma on Minikube.
```

# Cluster

## Overview

The folder contains various scripts involved in integration tests.

### Project structure

<!-- Update the folder structure each time you modify it. -->

The structure of the folder looks as follows:

```
  ├── cluster-integration               # Scripts for the cleanup of Google Cloud Platform (GCP) objects after completion of integration tests.             

>   ├── create-dns-entry.sh               # The script that creates a DNS entry on a GKE cluster for integration tests.

Reviewer 2: It creates a DNS entry in Google Cloud Platform, not GKE

>   ├── generate-self-signed-cert.sh      # The script for creating a self-signed certificate on a GKE cluster for integration tests.

Reviewer 2: It only creates certificates locally. Read description of script: https://github.com/kyma-project/test-infra/blob/master/prow/scripts/generate-self-signed-cert.sh

  ├── library.sh                        # The script to be used as an integral part of other scripts. For example, the "pipeline.sh" script uses it. With proper parameters defined, it authenticates you in GCP and sets up the Docker-in-Docker environment.

>   ├── pipeline.sh                       # The script responsible for building and testing a given Kyma component by running the respective "Makefile".

Reviewer 2: ```suggestion
  ├── pipeline.sh                       # The script responsible for building and testing a given Kyma component by running the respective "Makefile" target.
```
